Request/ArgumentValueResolver/DoctrineResolver.php

```diff
@@ -69,26 +111,26 @@ public function apply(Request $request, ParamConverter $configuration)
             // find by identifier?
         } elseif (false === $object = $this->find($class, $request, $options, $name)) {
             // find by criteria
+// FIXME: to be done elsewhere for expr for instance
+$method = $method->isArray() ? 'findBy' : 'findOneBy';
```

Comment:
`$method->isArray()` should probably be `$metadata->isArray()`

I think this case should only occur when guessing, otherwise it's simply a miss-configuration. If it's an array requested, it should probably never even hit `findOne` cases and thus never have an identifier in the URL. That means it can only be `findBy` and deserves its own case in this construction